tools/docker/mu-plugins/01-monorepo.php

```diff
@@ -39,7 +39,14 @@ class Monorepo {
 	 * Class constructor.
 	 */
 	public function __construct() {
-		$this->monorepo = '/usr/local/src/jetpack-monorepo/';
+		/**
+		 * Filter the monorepo path for development environments.
+		 *
+		 * @since $$next-version$$
```

Comment:
I don't think this will ever increment since it isn't in a project, but we can remove it later. Not going to block you for it.

Reply:
Good to know!